docs/src/pages/_app.page.tsx

```diff
@@ -8,6 +9,7 @@ import { Header } from '@/components/Layout/Header';
 import Script from 'next/script';
 import { baseTheme } from '../theme';
 import { useCustomRouter } from '@/components/useCustomRouter';
+import { awsExports } from '../data/amplifyMock/index.js';
```

Comment:
Does this import work since it's pointing to a `.js` file that doesn't exist? Either way, should be able to just write this as 
```suggestion
import { awsExports } from '../data/amplifyMock';
```
Since we are exporting from the _index.ts_ file